### PR TITLE
[DEV-5401] spending explorer periods

### PR DESCRIPTION
**Pull request by emily-brents**
**High level description:**

Warmfix adding support for defaulting to the latest period in the spending explorer rather than the latest quarter. Also adjusts some of the css.

**JIRA Ticket:**
[DEV-5401](https://federal-spending-transparency.atlassian.net/browse/DEV-5401)

**Mockup:**
https://bahdigital.invisionapp.com/share/C8IADXT2BD4#/screens/296036477

The following are ALL required for the PR to be merged:

Author:
- [X] Linked to this PR in JIRA ticket
- [ ] Scheduled demo including Design/Testing/Front-end OR Provided instructions for testing in JIRA and PR `if applicable`
- [X] Verified cross-browser compatibility: Chrome, Safari, Firefox, Internet Explorer, Edge
- [X] Verified mobile/tablet/desktop/monitor responsiveness
- [X] Verified that this PR does not create any *new* accessibility issues (via Axe Chrome extension)
- [X] Added Unit Tests for methods in Container Components, reducers, helper functions, and models `if applicable`
- [X] All `componentWillReceiveProps` and `componentWillMount` in relevant child/parent components/containers replaced with [future compatible life-cycle methods](https://reactjs.org/blog/2018/03/27/update-on-async-rendering.html) per [JIRA item 3277](https://federal-spending-transparency.atlassian.net/browse/DEV-3277)

Reviewer(s):
- [ ] Design review complete
- [ ] Code review complete

### CODE REVIEW

**Review by MaxwellKendall — CHANGES_REQUESTED** [review]
I'm gonna be out tomorrow but I think the main thing that needs to be looked at is this `mostRecentPeriod` fn. Not sure why we can't use [`getInitialPeriodByFy`](https://github.com/fedspendingtransparency/usaspending-website/blob/dev/src/js/redux/reducers/bulkDownload/bulkDownloadReducer.js#L14)?

Also the comments or the code need to be updated. I don't understand the logic behind how we're determining the active period. If for some reason we can't use the existing `fn` that selects the default period by FY in the bulkDownloadReducer, can we just update the comments so the logic is more readable?

Thanks for diving into the front end!

File: src/js/containers/explorer/detail/helpers/explorerQuarters.js [hunk 1]
```diff
@@ -150,3 +150,48 @@ export const defaultQuarters = () => {
     const current = mostRecentQuarter();
     return availableQuartersInFY(current.year);
 };
+
+
+export const mostRecentPeriod = () => {
+    // go back 30 days prior to today
+    const todayAdjusted = moment().add(3, 'months');
```
Comment (MaxwellKendall) [inline src/js/containers/explorer/detail/helpers/explorerQuarters.js:157]: Update the comment? Not sure what the logic is we're using here.

Comment (MaxwellKendall) [reply src/js/containers/explorer/detail/helpers/explorerQuarters.js:157]: I'm guesing this should be 

```suggestion
    const todayAdjusted = moment().subtract(30, 'days');
```

File: src/js/redux/reducers/explorer/explorerReducer.js [hunk 2]
```diff
@@ -14,12 +14,12 @@ export const ActiveScreen = new Record({
 });
 
 const initialQuarters = defaultQuarters();
-
+const initalPeriods = defaultPeriod();
```
Comment (MaxwellKendall) [inline src/js/redux/reducers/explorer/explorerReducer.js:17]: We should be able to just use [this fn in the bulkDownloadReducer](https://github.com/fedspendingtransparency/usaspending-website/blob/dev/src/js/redux/reducers/bulkDownload/bulkDownloadReducer.js#L14) for this.

Comment (emily-brents) [reply src/js/redux/reducers/explorer/explorerReducer.js:17]: We can't, the `getInitialPeriodByFy` function gets the latest period in the lastest closed quarter, which is not the latest closed period if that makes sense. Essentially the period will close before the quarter will, but that function is checking for closed quarters, which is what was causing the problem

File: src/js/containers/explorer/detail/helpers/explorerQuarters.js (hunk 1 above)
Comment (emily-brents) [reply src/js/containers/explorer/detail/helpers/explorerQuarters.js:157]: yeah that's what i get for copy pasting, updated the comment to be more clear

**Comment by emily-brents** [comment]
accidentally merged in dev

**Pull request closed without merging**